src/library-authoring/common/OrganizationDropdown.jsx

```diff
@@ -89,10 +89,11 @@ class OrganizationDropdown extends React.Component {
   }
 
   handleClick = (e) => {
-    const dropDownItems = this.getItems(e.target.value);
-    if (dropDownItems.length > 1) {
-      this.setState({ dropDownItems, icon: this.expandLessButton() });
+    let dropDownItems = this.getItems(e.target.value);
+    if (dropDownItems.length === 0) {
```

Comment:
```suggestion
    if (!dropDownItems.length) {
```
and further the same